components/script/dom/webgl_extensions/extensions.rs

```diff
@@ -105,6 +119,12 @@ impl WebGLExtensions {
         })
     }
 
+    pub fn is_enabled<T>(&self) -> bool
```

Comment:
nit: Indentation here looks weird (and below too, though that's pre-existing).

The standard rust formatting would be something like:

```rust
pub fn is_enabled<T>(&self) -> bool
where
    T: HeapSizeOf + JSTraceable + WebGLExtension + 'static,
{
    let name = ...;
    // ...
}